src/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs

```diff
@@ -166,7 +178,33 @@ private SyntaxNode Visit(SyntaxNode namedTypeNode, INamedTypeSymbol namedType)
             return namedTypeNode;
         }
 
-        private IEnumerable<INamespaceSymbol> EnumerateNamespaces(IAssemblySymbol assemblySymbol)
+        private SyntaxNode GenerateAssemblyAttributes(IAssemblySymbol assembly, SyntaxNode compilationUnit)
+        {
+            foreach (var attribute in assembly.GetAttributes())
```

Comment:
It probably makes sense to do attribute filtering here so that we can then exclude some assembly attributes that we don't want to emit.

Reply:
Ok.Updated